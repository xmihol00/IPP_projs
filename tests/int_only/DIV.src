<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode21">
  <instruction order="1" opcode="DEFVAR">
    <arg1 type="var">GF@x</arg1>
  </instruction>
  <instruction order="2" opcode="MOVE">
    <arg1 type="var">GF@x</arg1>
    <arg2 type="float">0x1.08p+3</arg2>
  </instruction>
  <instruction order="3" opcode="DEFVAR">
    <arg1 type="var">GF@y</arg1>
  </instruction>
  <instruction order="4" opcode="MOVE">
    <arg1 type="var">GF@y</arg1>
    <arg2 type="float">0x1.2p+2</arg2>
  </instruction>
  <instruction order="5" opcode="DEFVAR">
    <arg1 type="var">GF@res</arg1>
  </instruction>
  <instruction order="6" opcode="DIV">
    <arg1 type="var">GF@res</arg1>
    <arg2 type="var">GF@x</arg2>
    <arg3 type="var">GF@y</arg3>
  </instruction>
  <instruction order="7" opcode="WRITE">
    <arg1 type="var">GF@res</arg1>
  </instruction>
  <instruction order="8" opcode="WRITE">
    <arg1 type="string">\010</arg1>
  </instruction>
  <instruction order="9" opcode="DIV">
    <arg1 type="var">GF@res</arg1>
    <arg2 type="var">GF@y</arg2>
    <arg3 type="var">GF@x</arg3>
  </instruction>
  <instruction order="10" opcode="WRITE">
    <arg1 type="var">GF@res</arg1>
  </instruction>
  <instruction order="11" opcode="WRITE">
    <arg1 type="string">\010</arg1>
  </instruction>
</program>
