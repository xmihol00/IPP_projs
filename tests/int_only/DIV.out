0x1.d555555555555p+0
0x1.1745d1745d174p-1
